glib/Object.cs

```diff
@@ -390,12 +390,17 @@ protected Object (GType gtype)
 		[DllImport("glibsharpglue-2", CallingConvention=CallingConvention.Cdecl)]
 		static extern IntPtr gtksharp_object_newv (IntPtr gtype, int n_params, IntPtr[] names, GLib.Value[] vals);
 
-		protected virtual void CreateNativeObject (string[] names, GLib.Value[] vals)
+		protected void CreateNativeObject (string[] names, GLib.Value[] vals)
```

Comment:
it'd probably be better to not break it then. There isn't a compelling reason to break this :)